modules/tensor_mechanics/test/tests/shell/static/straintest_shear.i

```diff
@@ -0,0 +1,382 @@
+# Test for the shear stress and strain output for 2D planar shell with uniform mesh.
+# A  cantiliver beam of length 10 m  and cross-section 1.5 m x 0.1 m having
+# Young's Modulus of 5 N/mm^2 and poissons ratio of 0 is subjected to shear
+# displacement of 0.05 m at the free end.
+
+
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  nx = 4
+  ny = 1
+  xmin = 0.0
+  xmax = 10
+  ymin = 0.0
+  ymax = 1.5
+[]
+
+[Variables]
+  [disp_x]
+    order = FIRST
+    family = LAGRANGE
+  []
+  [disp_y]
+    order = FIRST
+    family = LAGRANGE
+  []
+  [./disp_z]
```

Comment:
how did this get missed?

Reply:
Removed ./ in all the test  input files.